tutorials/tutorial1_sinus-interactive.ipynb

```diff
@@ -191,7 +211,7 @@
    "source": [
     "Fortunately, decomon allows to find rigorous bounds of the model that will be true on the full input domain definition. \n",
     "\n",
-    "<img src=\"./data/decomon.jpg\" alt=\"Decomon!\" width=\"400\"/>\n",
+    "<img src=\"./data/decomon.png\" alt=\"Decomon!\" width=\"400\"/>\n",
```

Comment:
Same here, non decomon.png for now. (But if the idea is to introduce it later, we can let it like that but you should do it also for other notebooks in that case. Tutorial 3 still points to decomon.jpg somewhere)